app/presenters/concerns/assessment_tasks/assessment_detail_presenter.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module AssessmentTasks
+  extend ActiveSupport::Concern
+
+  class AssessmentDetailPresenter < PlanningApplicationPresenter
```

Comment:
Does this need to inherit everything from `PlanningApplicationPresenter`? Instead you could pull anything you need in that class into a concern and include it here? Otherwise this class has access to all the methods in `StatusPresenter`, `ProposalDetailsPresenter`, `ValidationTasksPresenter` and `AssessmentTasksPresenter`, since they're all included in `PlanningApplicationPresenter`...which seems a bit mad!

Reply:
That said, maybe this just means we're due to a bit of a presenter tidy up generally, which could wait for another day 😁

Reply:
Good point, we certainly don't want that! I'll see what I can do

Reply:
On second thoughts i'm gonna make a card for this as it might bloat this PR

Reply:
Yes, I think that's a good idea! More of a nice to have refactor than a requirement for this piece of work ☺️